crates/unleash-edge-frontend-api/src/querystring_extractor.rs

```diff
@@ -2,22 +2,19 @@ use axum::extract::FromRequestParts;
 use axum::http::request::Parts;
 use serde::de::DeserializeOwned;
 use serde_qs::Config;
-use unleash_edge_appstate::AppState;
 use unleash_edge_types::errors::EdgeError;
 
 /// Wrapper type like QsQuery, but with custom config.
 pub struct QsQueryCfg<T>(pub T);
 
-impl<T> FromRequestParts<AppState> for QsQueryCfg<T>
+impl<S, T> FromRequestParts<S> for QsQueryCfg<T>
```

Comment:
Similar dance to earlier, this is now just generic over anything Send+Sync so we no longer need AppState here